x/smart/README.md

```diff
@@ -0,0 +1,134 @@
+# Smart Dialer
+
+The **Smart Dialer** is a dialer that **automatically chooses the best way to connect** to the internet from a list of configured options.
```

Comment:
"Best" is not really well defined, and we don't really do that.
We just pick a way that ensures we can successfully connect to a list of given services.

I think saying "connect to the internet" also conveys something else.